### PR TITLE
docs: describe the release workflow in the contributing guide

### PR DESCRIPTION
**Pull request by ZauberNerd**
<details>
<summary>Bors merge bot cheat sheet</summary>

We are using [bors-ng](https://github.com/bors-ng/bors-ng) to automate merging of our pull requests. The following table provides a summary of commands that are available to reviewers (members of this repository with push access) and delegates (in case of `bors delegate+` or `bors delegate=[list]`).

| Syntax | Description |
| --- | --- |
| bors merge | Run the test suite and push to master if it passes. Short for "reviewed: looks good." |
| bors merge- | Cancel an r+, r=, merge, or merge= |
| bors try | Run the test suite without pushing to master. |
| bors try- | Cancel a try |
| bors delegate+ | Allow the pull request author to merge their changes. |
| bors delegate=[list] | Allow the listed users to r+ this pull request's changes. |
| bors retry | Run the previous command a second time. |

This is a short collection of opinionated commands. For a full list of the commands read the [bors reference](https://bors.tech/documentation/).

</details>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,23 +22,56 @@ First off, we really value your contribution. Thanks for wanting to help us with
...
 We are using the `master` branch as our **staging branch** and we have a maximum of two active **release branches** at a time. The release branches follow the naming convention of `vN.x` (where `N` is the major version number of that release).
 
 All PRs (except for PRs generated by renovate or release PRs) must be made against `master` and include in their description, and through a label, for which release they are intended.
 
-We have appointed branch maintainers for each of the release branches, who will take care of cherry-picking the required changes.
+### Releasing a new version
+
+Hops releases are fully automated through the release-bot.
+
+Each PR must have at least one release label, indicating which version it targets (":package: master", ":package: v14", etc..). Once the PR has been merged the release-bot will try to cherry-pick it into the respective release branch and notify you if there has been a merge conflict. Then the release-bot creates a "Release PR", which - once merged - will trigger an automatic publish to npm.
```
Comment (herschel666) [inline CONTRIBUTING.md:35]:
```diff
- to cherry-pick it into the respective release branch
+ to cherry-pick it onto the respective release branch
```

**Review by herschel666 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ZauberNerd) [reply CONTRIBUTING.md]: Thanks, updated.

**Comment by ZauberNerd** [comment]
bors merge

**Comment by ZauberNerd** [comment]
bors retry

**Pull request merged**